src/yoroi-wallets/cardano/shelley/__snapshots__/transactionCache.test.ts.snap

```diff
@@ -111,7 +117,22 @@ Object {
 }
 `;
 
-exports[`transactionCache create loads from storage 3`] = `Object {}`;
+exports[`transactionCache create loads from storage 3`] = `
+Object {
```

Comment:
why did this change?

Reply:
To test the changes in the variable perRewardAddressCertificates

Reply:
Also, transactionCache.test.ts and transactionManager.test.ts are using the same mocks now.